### DISCUSSION TITLE
Can you specify a certain branch?

### DISCUSSION THREAD

**Discussion by PEConn** [Vinzent03/obsidian-git#372] [Q&A]
Hey,

I'm using obsidian-git to sync between desktop and Android. Occasionally there's a merge conflict, and there doesn't seem to be a way to deal with these on Android - when I next Backup from Android, the Android version of the file overwrites anything else. (IIUC, not being able to deal with merges is a limitation of `isomorphic-git`.)

One thing that would help, is if I could specify a different branch for my Android phone, then manually merge the changes in on desktop.

Is there any way to do this?

Thanks,
Peter C

**Comment by Vinzent03** [comment]
I will check the merge conflict issue in the next days, but there are commands to create and switch branches. You can't merge them via Obsidian, but of course via terminal.

**Comment by PEConn** [comment]
Oh right, it's in the command palette! I was looking for it in settings. Thanks that helps so much!

So a bit more context on the merge issues - basically what happens is:

1. I edit a file on desktop and backup.
2. Before pulling, I edit a file on Android.
3. Android alerts me there is a merge conflict and it opens a "Merge conflict view" or something that tells me which files there is a conflict in.
4. I'm not sure how to resolve it.
5. I backup from Android.

This results in a commit that "adds" the entire file (the version of it that was on Android), which is pretty weird because the file already existed in the repo.

That means that if I do `git show <file>` it just shows the entire thing as added, but if I do `git diff <previous commit> -- <file>`, then it shows the actual changes that were made.
